tests/Fixer/Operator/TernaryToNullCoalescingFixerTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Operator;
+
+use PhpCsFixer\Test\AbstractFixerTestCase;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ *
+ * @internal
+ */
+final class TernaryToNullCoalescingFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @dataProvider provideCases
+     * @requires PHP 7.0
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideCases()
+    {
+        return array(
+            array('<?php $x = isset($a) ? $a[1] : null;'),
+            array('<?php $x = isset($a) and $a ? $a : "";'),
+            array('<?php $x = "isset($a) ? $a : null";'),
+            array('<?php $x = isset($a) ? $$a : null;'),
+            array('<?php $x = isset($a) ? "$a" : null;'),
+            array('<?php $x = isset($a) ?: false;'),
+            array('<?php $x = $a ? $a : isset($b) ? $b : isset($c) ? $c : "";'),
+            array('<?php $x = $y ?? isset($a) ? $a : null;'),
+
+            array(
+                '<?php $x = $a ?? null;',
+                '<?php $x = isset($a) ? $a : null;',
+            ),
+            array(
+                '<?php $x = $a ?? 1; $y = isset($b) ? "b" : 2; $x = $c ?? 3;',
+                '<?php $x = isset($a) ? $a : 1; $y = isset($b) ? "b" : 2; $x = isset($c) ? $c : 3;',
+            ),
+            array(
+                '<?php $x = $a[ count ($b[ "c"  ])]   ?? null;',
+                '<?php $x = isset   (  $a[count($b["c"])]) ?$a[ count ($b[ "c"  ])]   : null;',
+            ),
+            array(
+                '<?php $x = $a ?? $b[func(1, true)];',
+                '<?php $x = isset($a) ? $a : $b[func(1, true)];',
+            ),
+            array(
+                '<?php $x = $a ?? ($b ?? "");',
+                '<?php $x = isset($a) ? $a : (isset($b) ? $b : "");',
+            ),
+            array(
+                '<?php $x = ($a ?? isset($b)) ? $b : "";',
+                '<?php $x = (isset($a) ? $a : isset($b)) ? $b : "";',
+            ),
+            array(
+                '<?php $x = $a ?? isset($b) ? $b : isset($c) ? $c : "";',
+                '<?php $x = isset($a) ? $a : isset($b) ? $b : isset($c) ? $c : "";',
```

Comment:
As of yet this case would be fixed like:
```php
<?php $x = $a /*c*/ ?? /*d*/ isset($b) /*e*/ ? /*f*/ $b /*g*/ : /*h*/ isset($c) /*i*/ ? /*j*/ $c /*k*/ : /*l*/ "";
```
Syntactically ok, but losing `/*a*/` and `/*b*/`: where should they go?
Something like this would be ok?

```php
<?php $x = /*a*/ /*b*/ $a /*c*/ ?? /*d*/ isset($b) /*e*/ ? /*f*/ $b /*g*/ : /*h*/ isset($c) /*i*/ ? /*j*/ $c /*k*/ : /*l*/ "";
```